templates/person/detail.html

```diff
@@ -266,7 +273,16 @@ <h3 id="subordinates"><i class="fa fa-fw fa-share-alt fa-rotate-90"></i> {% tran
                         window.location = node.url;
                     }
                 });
-                
+
+                // Transform cursor (from hand into pointed finger and back again)
+                network.on("hoverNode", function (params) {
```

Comment:
Any chance we could do this with a CSS `:hover` selector instead?

Reply:
Yep, that was my first attempt...but I had some trouble identifying the classes to select in the CSS (dev tools does not allow you to inspect individual vis.js nodes). This, again, was a suggestion in the official vis.js examples, combined with some magic on stackoverflow.

https://github.com/almende/vis/blob/bc2bc581aaa16905f2ee15b26d3915648c34cc54/examples/network/other/cursorChange.html#L114
https://stackoverflow.com/questions/40532596/change-mouse-cursor-over-vis-js-network-node